### PR TITLE
Only combat defib can disarm

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

Only combat defibs can disarm/knockdown, regular defibs can still harm when safeties are disabled using EMP. 

## Why It's Good For The Game

No more instant knockdown/disarm from defibs.
Doctors can still powergame with their 1u uranium 1u iron mixes to kill people.
Changelings and bloodbrothers can still rely on crutches and remain underpowered because other items exist.

## Changelog 
:cl:
balance: Only combat defibs can use disarm intent.
/:cl:

### CODE REVIEW

File: code/game/objects/items/defib.dm [hunk 1]
```diff
@@ -473,9 +473,7 @@
 			M.emote("scream")
 
 /obj/item/twohanded/shockpaddles/proc/do_disarm(mob/living/M, mob/living/user)
-	if(req_defib && defib.safety)
-		return
-	if(!req_defib && !combat)
+	if(!combat)
```
Comment (ghost) [inline code/game/objects/items/defib.dm:476]: >!req_defib || req_defib

Negates itself so removed.

**Comment by TotallyNotChief** [comment]
no

**Comment by nikk735** [comment]
cringe af

**Comment by ghost** [comment]
maybe you should make emp-ed defibs only gives stamina damage only while emagged defibs still have the old stun?

**Review by PowerfulBacon — APPROVED** [review]
Like I said on the other PR

Big + from stopping the power-gaming CMO from being able to have a portable instant stun device. However on the other hand, it limits the already limited potential of antagonists that do not have access to traitor gear (Blood brothers etc.). Wish there was a way to get the best of both worlds

**Comment by VictorVow** [comment]
Make it like a stun prod, a good amount of stamina damage but no knockdown. Voila.

**Pull request closed without merging**